src/views/login/screen3.tsx

```diff
@@ -131,10 +131,10 @@ export default class LoginScreen2 extends Component<{}, LoginScreen2State> {
               behavior="position"
             >
               <View style={styles.titleContainer}>
-                <View style={{ flexDirection: 'row' }}>
+                <View>
                   <Text style={styles.titleText}>BEAUX</Text>
                 </View>
-                <View style={{ marginTop: -10, marginLeft: 10 }}>
+                <View style={{ marginLeft: 10 }}>
```

Comment:
Are you sure that the "Beaux Voyages" is meant to be center-aligned with padding in between them? Cuz looking at the logo design, I feel like the alignment of the titleText was left like that on purpose (otherwise someone would have changed it a long time ago as it was created 3 years back😄 )

Reply:
I have done it so that it looks the same on web as it was looking on mobile devices. Thanks!